packages/backend/src/rapyd/documents/documents.service.ts

```diff
@@ -0,0 +1,48 @@
+import logger from '../../utils/logger'
+import { Request, Response } from 'express'
+import { BaseResponse } from '../../shared/models/BaseResponse'
+import { makeRapydGetRequest } from '../utills/request'
+import { User } from '../../user/models/user'
+
+const log = logger('rapydService')
+
+export const getDocumentTypes = async (
+  req: Request,
+  res: Response<BaseResponse>
+) => {
+  try {
+    const { email } = req.user as User
```

Comment:
req.user also has the id. Maybe use that to query the db for the country retrieval, since the id being the primary key is also indexed, the email is not.